File: javascript/7kyu/Thinking & Testing Uniq or not Uniq.js
// https://www.codewars.com/kata/56d949281b5fdc7666000004

function testit(a,b){
    a = [...new Set(a)];
    b = [...new Set(b)];
    return a.concat(b).sort(function(c,d){return c - d;});
}

File: java/7kyu/Thinking & Testing Uniq or not Uniq.java
// https://www.codewars.com/kata/56d949281b5fdc7666000004

import java.util.stream.IntStream;
import java.util.Arrays;

public class Kata {
  public static int[] testit(int[] a, int[] b){
    IntStream aStr = Arrays.stream(a).distinct();
    IntStream bStr = Arrays.stream(b).distinct();
    return IntStream.concat(aStr, bStr).sorted().toArray();
  }
}